packages/react-meteor-state/README.md

```diff
@@ -0,0 +1,4 @@
+meteor/react-meteor-state
+=========================
+
+A state hook, with an API similar to React's useState, which provides data persistence between page reloads using hte Meteor reload package.
```

Comment:
typo in `hot`

Reply:
That whole readme needs to be written. :)